crates/rpc/rpc-builder/src/auth.rs

```diff
@@ -372,4 +452,22 @@ impl AuthServerHandle {
             .await
             .expect("Failed to create ws client")
     }
+
+    /// Returns an ipc client connected to the server.
+    pub async fn ipc_client(&self) -> jsonrpsee::async_client::Client {
+        IpcClientBuilder::default()
```

Comment:
this function needs to be feature gated to unix
and should return `Option<IpcClient>`